system-tests/jobs/test-jobs.js

```diff
@@ -7,7 +7,7 @@ describe("Jobs", function() {
     });
   });
 
-  it("Create a simple job", function() {
+  it("should create a simple job", function() {
```

Comment:
@weblancaster using `should` here makes perfect sense as it aligns the system tests with our unit and integration tests. If you feel we should abandon the pattern, please add an item to our team sync agenda to discuss the topic.